app/src/main/java/ch/epfl/sweng/swenggolf/preference/ListPreferenceAdapter.java

```diff
@@ -33,21 +32,21 @@
     private ArrayList<User> mDataset;
 
     public static final User[] usersInitial = {
```

Comment:
This variable is `static final`. It would follow more closely the style rules if it were written in uppercase.